src/trace-plugin-interface.js

```diff
@@ -111,9 +111,10 @@ var nullSpan = {
  * plugin themselves
  * TODO(kjin): Should be called something else
  */
-function PluginAPI(agent) {
+function PluginAPI(agent, pluginName) {
   this.agent_ = agent;
   this.logger_ = agent.logger;
+  this.pluginName_ = pluginName;
```

Comment:
It isn't, but I wanted to include it anyway to make the follow-up change that adds it to log messages more straightforward. In fact, I think most of that is small enough that I could do it in this PR.